src/parser/bison_parser.y

```diff
@@ -557,10 +566,14 @@ column_type:
 	|	INTEGER { $$ = ColumnType{DataType::INT}; }
 	|	LONG { $$ = ColumnType{DataType::LONG}; }
 	|	FLOAT { $$ = ColumnType{DataType::FLOAT}; }
+	|   DECIMAL '(' INTVAL ',' INTVAL ')' { $$ = ColumnType{DataType::FLOAT}; }
```

Comment:
I think we should add a complete decimal support (including decimal literals, and optional precision and scale values) or leave it out.

Reply:
I think it should be ```DataType::DECIMAL```

Reply:
I agree that it should be supported either as a whole or not at all, but supporting it as a whole would exceed the seminar's scope and the time left. @mweisgut should we delete it then, or should we leave it as is, with a comprehensive parsing warning once it reaches the sql-translator.

Reply:
The completeness here refers only to the SQL parser, not to Hyrise: if we parse the column type DECIMAL, then this type has one parameter (precision) or two parameters (precision, scale). This information should still be present in the abstract syntax tree after parsing. However, in the SQL translator in Hyrise we continue working with Float when finding the DataType DECIMAL in the syntax tree. Similarly, we also want to parse the optional precision parameter for the column type TIME and store this information in the syntax tree. Inside Hyrise, however, we can then work with a different data type.

See
- https://www.postgresql.org/docs/9.1/datatype-numeric.html
- https://www.postgresql.org/docs/9.1/datatype-datetime.html

Reply:
Has been resolved in https://github.com/hyrise/sql-parser/pull/177/commits/67e6d65ecc5ae7d30f6738aa103971e4afa5cb94